examples/fetch/src/post.rs

```diff
@@ -0,0 +1,83 @@
+//! Fetch POST example.
+
+use seed::{prelude::*, *};
+
+#[derive(serde::Serialize, Default)]
+pub struct Form {
+    name: String,
+}
+
+#[derive(Default)]
+pub struct Model {
+    form: Form,
+    message: Option<String>,
+}
+
+pub enum Msg {
+    NameChanged(String),
+    Submit,
+    Submited,
+    SubmitFailed(String),
+}
+
+pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
+    match msg {
+        Msg::NameChanged(name) => model.form.name = name,
+        Msg::Submit => {
+            orders.skip(); // No need to rerender
+
+            let token = "YWxhZGRpbjpvcGVuc2VzYW1l";
+            // Created outside async block because of lifetime reasons
+            // (we can't use reference to `model.from` in async
+            // function).
```

Comment:
Is it on the standalone line because of GitHub weird line breaks or Rust formatter?